wiki/Storyboarding/Storyboarder/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - SBer
+---
+
+<!-- not sure if we need this -->
```

Comment:
don't think there much we can add since the term is pretty self explanatory

then again i'm not really a storyboarder myself or know much about sbs in general tbh